actions/issue/add-to-project/entrypoint

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
@arjun024 would you mind also adding:

```
shopt -s inherit_errexit
```

It will ensure that the script fails with error when a subshell (`$()`) exits with non-zero code.

Reply:
Done. Thanks!